src/components/line-tabs/LineTabs.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { SLineTabs } from './styles/SLineTabs';
+
+export interface ILineTabsProps {
+  hover?: boolean;
```

Comment:
hover event should only be handled by the browser, no js. Is there a reason for this?

Reply:
Same explanation goes for here.